integration/__tests__/cluster-pages.tests.ts

```diff
@@ -323,148 +39,390 @@ utils.describeIf(minikubeReady(TEST_NAMESPACE))("Minikube based tests", () => {
     await frame.waitForSelector(`.Menu >> text="Remove"`);
   });
 
-  it("should navigate around common cluster pages", async () => {
-    for (const test of commonPageTests) {
-      if (isTopPageTest(test)) {
-        const { href, expectedText, expectedSelector } = test.page;
-        const menuButton = await frame.waitForSelector(`a[href^="${href}"]`);
+  it(
```

Comment:
Can we also standardize our formatting rules ASAP. All these changes which your prettier config is introducing is making diffs much larger and is not consistent with the majority of the code base.

Reply:
You have my axe!

Diffs and inconsistency is one thing, but not being able to properly use one-truth formatting in development is a real downer.

Reply:
I agree with @Nokel81. Until prettier (or another tool) is integrated I think developers should disable automatic formatting changes like these.

Reply:
Yep, we've removed these type of changes previously. Couple slipped in, so we'll revert those.

Reply:
Done